niceman/distributions/conda.py

```diff
@@ -92,13 +92,43 @@ def install_packages(self, session=None):
             Environment sub-class instance.
         """
 
-        # TODO: Need to figure out a graceful way to install conda before we can install packages here.
-        # for package in self.provenance['packages']:
-        #     session.add_command(['conda',
-        #                            'install',
-        #                            package['name']])
+        # TODO: Install Conda
+#        print("We got" + str(environment))
+        # Loop through non-root packages, creating the conda-env export
+        # file to import
+        for env in self.environments:
+            if env.path == self.path:
+                continue
+            export_contents = self.create_conda_export(env)
+            print ("EXPORT\n" + export_contents)
```

Comment:
`lgr.debug` or it was intended to be displayed to the user?

Reply:
It's for me until I get things working :)